src/src.wake

```diff
@@ -35,19 +35,28 @@ def src (dir: String) (variant: Pair String String) (internal: List (Pair String
     def externalLib =
         flattenSysLibs externalLibs
 
-    def ym4s =
+    require Pass ym4s =
         sources here `.*\.y\.m4`
 
     require Pass parsers =
         map (parserGen variant) ym4s
         | findFail
 
+    require Pass versionHeader =
+        version_h Unit
+    require Pass headerPaths =
+        def searchDirs =
+            srcDir,
+            vendorDir,
+            Nil
+        searchDirs
+        | findFailFn (sources _ `.*\.h`)
+        | rmap flatten
```

Comment:
You used this pattern in a few places. It's correct, but baroque. I wonder if there is a better pithy and straight-forward way.

Reply:
Probably just two `require`s for direct calls to `sources` would be more readable.

Reply:
Yeah, probably.  When there's half a dozen `sources` calls I start looking for ways of removing the boilerplate, but it's definitely baroque.